dev/config/backend_next/.test_env

```diff
@@ -0,0 +1,41 @@
+MONGODB_URI = mongodb://mongodb:27017/test_be_next
+EXPRESS_SESSION_SECRET = a_scicat_secret
+JWT_SECRET = a_scicat_secret
+PORT = 3000
+HTTP_MAX_REDIRECTS = 5
+HTTP_TIMEOUT = 5000
+JWT_EXPIRES_IN = 3600
+LOGGING_DISABLED = True
+# NOTE = The SITE variable is set to PSI because of the PublishedData test.
+# Testing /register endpoint expects to have it as PSI for that specific scenario.
+# Old backend was modifying this at runtime but with NestJS we need to investigate little bit more if there is a bit better and more elegant solution.
+SITE = PSI
+PID_PREFIX = "API_testing"
+DOI_PREFIX = DOI.SAMPLE.PREFIX
+METADATA_KEYS_RETURN_LIMIT = 100
+METADATA_PARENT_INSTANCES_RETURN_LIMIT = 100
+ADMIN_GROUPS = "admin,adminingestor"
+DELETE_GROUPS = "archivemanager"
+CREATE_DATASET_GROUPS = group1,group2,group3
+CREATE_DATASET_WITH_PID_GROUPS = "group2"
+CREATE_DATASET_PRIVILEGED_GROUPS = "datasetIngestor,group3"
+CREATE_JOB_GROUPS = group1,group2
+UPDATE_JOB_GROUPS = group1
+ACCESS_GROUPS_STATIC_VALUES = "ess"
+PROPOSAL_GROUPS = "proposalingestor"
+SAMPLE_PRIVILEGED_GROUPS = "sampleingestor"
+SAMPLE_GROUPS = "group1"
+DATASET_CREATION_VALIDATION_ENABLED = true
+DATASET_CREATION_VALIDATION_REGEX = "^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$"
+ES_HOST = https://localhost:9200
```

Comment:
are the ES options relevant? I thought there was no ES enabled. If so, could we please remove them?